doc/ref_cert/README.md

```diff
@@ -18,24 +20,24 @@ If there is no clear traceability and strong links between these 3 components, t
 * Reduce ambiguity in testing, requirements, and conformance
 
 
-## Requirement Principles
+### Requirement Principles
 Requirements Principles can be found in the [RM Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0)
 
-## Testing Principles
+### Testing Principles
 * There must be traceability between test cases and requirement being validated
 * Failures should provide additional content to inform the user where or how the requirement was violated (ex: which file or resource violated the requirement).  Put another way, don’t require the user to read the test to understand what went wrong
```

Comment:
Does `ex:` mean `example` or `excluding`.  Suggest either using `e.g.` or `excluding`.

Reply:
I've created a separate pr to fix this in #2197